charts/fluent-operator/values.yaml

```diff
@@ -78,11 +78,11 @@ fluentd:
     tag: "v1.14.4"
   replicas: 1
   forward:
-    host: "kubesphere-logging-system.svc"
+    host: "fluent.svc"
```

Comment:
fluent.svc is a valid service? @zhu733756 @wenchajun

Reply:
Yes, if fluentd is enabled, fluent-operator will create an svc based on the namespace it is in.

Reply:
in fluent.svc, fluent is the namespace, where is the service name?

Reply:
fluent-operator automatically creates a svc when `fluentd` is enabled, see this : https://github.com/fluent/fluent-operator/blob/master/pkg/operator/service.go#L26
Because the first part of svc uses the name of fluentd, in helm we split it into two parts.
https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/templates/fluentbit-output-forward.yaml#L13

Reply:
That's not host, it's domain actually

Reply:
> That's not host, it's domain actually

Thanks,fixed.

Reply:
add a host field here is confusing.
Maybe we can delete it, and just use  
    host: {{ .Values.fluentd.name }}.<namespace>.svc

Reply:
> add a host field here is confusing. Maybe we can delete it, and just use host: {{ .Values.fluentd.name }}..svc in the template

Its domain will be different depending on the namespace, is it better not to take a fixed value? I have changed the name in value from `host` to `domain`.

Reply:
I mean we don't need any field here.
Simply use host: {{ .Values.fluentd.name }}.< namespace >.svc in the template